netdata.spec.in

```diff
@@ -21,9 +21,6 @@ AutoReqProv: yes
 
 # Disable go.d.plugin build on outdated golang distros
 %if 0%{?centos_ver:1}
-%if 0%{?centos_ver} >= 8 && 0%{!?almalinux_ver:1} && 0%{!?rocky_ver:1}
```

Comment:
- I think you need to remove one endif too
- And probably add to the comment `(golang 1.21+ is required)`

Reply:
yes yes, my bad

Reply:
> And probably add to the comment (golang 1.21+ is required)
give me a sec to online debug it via actions first